services/QuillLMS/spec/services/vitally_integration/serialize_vitally_sales_organization_spec.rb

```diff
@@ -0,0 +1,313 @@
+        end
+
+        it 'in the previous school year' do
+          expect(SerializeVitallySalesOrganization.new(district).data[:traits][:activities_completed_last_year]).to eq(3)
+        end
+
+        it 'for all time' do
+          expect(SerializeVitallySalesOrganization.new(district).data[:traits][:activities_completed_all_time]).to eq(5)
+        end
+      end
+
+      context 'gets the number of activites completed per student' do
+
+        it 'in this school year' do
+          expect(SerializeVitallySalesOrganization.new(district).data[:traits][:activities_completed_per_student_this_year]).to eq(1.0)
+        end
+
+        it 'in the previous school year' do
+          expect(SerializeVitallySalesOrganization.new(district).data[:traits][:activities_completed_per_student_last_year]).to eq(1.5)
+        end
+
+        it 'for all time' do
+          expect(SerializeVitallySalesOrganization.new(district).data[:traits][:activities_completed_per_student_all_time]).to eq(2.5)
+        end
+      end
+
+      context 'gets the number of active students' do
+
+        it 'in this school year' do
+          expect(SerializeVitallySalesOrganization.new(district).data[:traits][:active_students_this_year]).to eq(2)
+        end
+
+        it 'in the previous school year' do
+          expect(SerializeVitallySalesOrganization.new(district).data[:traits][:active_students_last_year]).to eq(2)
+        end
+
+        it 'for all time' do
+          expect(SerializeVitallySalesOrganization.new(district).data[:traits][:active_students_all_time]).to eq(2)
+        end
+      end
+    end
+  end
+
+  context 'benchmarking', :benchmarking do
+    let!(:district) { create(:district) }
+    let!(:connect_activity) { create(:connect_activity) }
+    let!(:last_school_year) { Time.zone.today - 1.year }
+    let!(:schools) { create_list(:school, 80, district: district)}
+
+    it 'run benchmarking on a district with 80 schools and 10000 activities' do
+      schools.each do |school|
+        teacher = create(:teacher, school: school)
+        classroom = create(:classroom)
+        classroom_teacher = create(:classrooms_teacher, user: teacher, classroom: classroom)
+        diagnostic = create(:diagnostic_activity)
+        unit = create(:unit, activities: [diagnostic])
+        students = create_list(:student, 25, student_in_classroom: [classroom], last_sign_in: Time.zone.today)
+        classroom_unit = create(:classroom_unit, classroom: classroom, unit: unit, assigned_student_ids: students.map(&:id))
+        students.each do |student|
+          create_list(:activity_session, 5, state: 'finished', completed_at: Time.current, user: student, classroom_unit: classroom_unit, activity: diagnostic)
+        end
+      end
+
+      runtime = Benchmark.realtime { expect(SerializeVitallySalesOrganization.new(district).active_students).to eq(2000) }
```

Comment:
Note, for future PRs: You don't need to wrap the thing you are benching in an expectation.